selfdrive/test/process_replay/process_replay.py

```diff
@@ -344,6 +344,7 @@ def get_car_params_callback(rc, pm, msgs, fingerprint):
   if fingerprint:
     CarInterface, _, _, _ = interfaces[fingerprint]
     CP = CarInterface.get_non_essential_params(fingerprint)
+    CP, CP_SP = CarInterface.get_non_essential_params_sp(CP, fingerprint)
```

Comment:
Why do you overwrite CP? if the plan was to overwrite it, then maybe just remove the line above `CP = CarInterface.get_non_essential_params(fingerprint)` and only define it as part of `CP, CP_SP = CarInterface.get_non_essential_params_sp(CP, fingerprint)` and ofc you don't pass yourself `CP` as param

Reply:
We need `CP` to get passed into `get_non_essential_params_sp`, then gets consumed by `get_params_sp` as it needs the stock `CP` as one of the arguments.

We don't necessarily _need_ to return/overwrite CP in this test specifically, but we do return the tuple of `CP` and `CP_SP` from `get_params_sp`.